app/components/pagy_paginator_component/pagy_paginator_component.html.erb

```diff
@@ -0,0 +1,25 @@
+<div id="<%= id %>" class="<%= container_classes %>">
```

Comment:
OK, nitpicky but I think it reads much nicer.
```suggestion
<%= tag.div id: id, class: container_classes do %>
  # ...
<% end %>
```

Reply:
Agreed